.github/workflows/bot.yml

```diff
@@ -361,21 +361,24 @@ jobs:
           - scalaProfile: "scala-2.12"
             sparkProfile: "spark3.4"
             sparkModules: "hudi-spark-datasource/hudi-spark3.4.x"
+          - scalaProfile: "scala-2.13"
```

Comment:
Java tests in GH CI are split to two jobs: `test-spark-java17-java-unit-tests` and `test-spark-java17-java-functional-tests`.  `spark4` profile should also be added to `test-spark-java17-java-unit-tests`.